release/mixer.sh

```diff
@@ -54,9 +54,9 @@ build_update() {
 
     section "Build 'Update' Content"
     if ${MIN_VERSION:-false}; then
-        sudo -E mixer --native build update --skip-format-check --min-version="${mix_ver}"
+        sudo -E mixer --native build update --skip-format-check --min-version="${mix_ver}" "${SIGN_VAR}"
     else
-        sudo -E mixer --native build update --skip-format-check
+        sudo -E mixer --native build update --skip-format-check "${SIGN_VAR}"
```

Comment:
This may actually do:

```bash
build_update() {
    local build_args=( "--skip-format-check" f)

    # we dont want the quotes here
    if ${MIN_VERSION:-false}; then
        build_args += ( "--min-version=${mix_ver}" )
    fi

    if [[ -n "${MOM_SIGN_CMD}" ]]; then
        build_args += ( "--no-signing" )
    fi
....
    sudo -E mixer --native build update "${build_args[@]}" # it may be that this need quotes disabled

   # than we have some options:
   # 1) ${MOM_SIGN_CMD:-:} <path to mom> <maybe path to sign mom> # if mom_sign_cmd undefined, it will use ':' that is shell no-op built in
   # 2)if [[ -n "${MOM_SIGN_CMD}" ]] ; then sign_mom <path> <path>; fi # this allow us to log something if custom signing will be used. But true is that we could also log it in prologue.sh and be silent here
   # 3) same if as 2) and but: ${MOM_SIGN_CMD} <path> <path>
   # I'm more included to 1), but depending on how logging looks on final version, 2) may be better.
}
```